src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/PaymentMessageHandler.java

```diff
@@ -54,4 +57,28 @@ public CreatePaymentResponse handlePaymentMessage(PaymentMessage paymentMessage)
         return paymentResult;
     }
 
+
+    public void updatePaymentCaseReference(UpdatePaymentMessage paymentMessage) {
+
+        CaseReferenceRequest request = new CaseReferenceRequest(paymentMessage.newCaseRef);
+
+        log.info(
+            "Sending payment update case reference request, envelope id {}, case ref {}, exception ref {}",
+            paymentMessage.envelopeId,
+            request.ccdCaseNumber,
+            paymentMessage.exceptionRecordRef
+        );
+
+        ResponseEntity<?> response = payHubClient.updateCaseReference(
+            authTokenGenerator.generate(),
+            paymentMessage.exceptionRecordRef,
+            request
+        );
+
+        log.info("Payment update response from PayHub, envelope id {}, http status {} ",
```

Comment:
worth adding `:` in the message for better readability imho.

Reply:
added